### PR TITLE
Jan Release readme fix/ changes

### PR DESCRIPTION
**Pull request by hemanttanwar**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/src/samples/README.md [hunk 1]
```diff
@@ -13,9 +13,6 @@ Getting started explained in detail [here][SDK_README_GETTING_STARTED].
 
 For details on including this dependency in other build tools (Gradle, SBT, etc), refer [here](https://search.maven.org/artifact/com.azure/azure-core).
 
-## How to run
-These sample can be run in your IDE with default JDK.
-
```
Comment (hemanttanwar) [inline sdk/storage/azure-storage-blob/src/samples/README.md]: Not needed, nothing in here as instructions.

File: sdk/storage/azure-storage-queue/src/samples/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-## Azure Azure Storage Queue Samples client library for Java
```
Comment (hemanttanwar) [inline sdk/storage/azure-storage-queue/src/samples/README.md]: Azure two times.

File: sdk/appconfiguration/azure-data-appconfiguration/src/samples/README.md [hunk 3]
```diff
@@ -14,9 +14,6 @@ This document explains samples and how to use them.
 ## Key concepts
 Key concepts are explained in detail [here][SDK_README_KEY_CONCEPTS].
```
Comment (kaerm) [inline sdk/appconfiguration/azure-data-appconfiguration/src/samples/README.md]: In line 7 should be `- azure-app-configuration`

File: sdk/storage/azure-storage-blob-batch/src/samples/README.md [hunk 4]
```diff
@@ -1,20 +1,23 @@
-
-## Azure Azure Storage Blob Batch Samples client library for Java
+---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-blob-storage
```
Comment (kaerm) [inline sdk/storage/azure-storage-blob-batch/src/samples/README.md]: I'd also add one more line for ` - azure-storage`

File: sdk/storage/azure-storage-blob/src/samples/README.md [hunk 5]
```diff
@@ -1,5 +1,14 @@
...
-## Azure Azure Storage Blob Samples client library for Java
+---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-blob-storage
```
Comment (kaerm) [inline sdk/storage/azure-storage-blob/src/samples/README.md]: I'd also add one more line for ` - azure-storage`

File: sdk/storage/azure-storage-blob/src/samples/README.md [hunk 6]
```diff
@@ -27,7 +33,6 @@ These sample can be run in your IDE with default JDK.
 6. [Azure Identity Examples][samples_identity]: Use `DefaultAzureCredential` to do the authentication.
 
 ## Troubleshooting
-### General
 When interacting with blobs using this Java client library, errors returned by the service correspond to the same HTTP
 status codes returned for [REST API][error_codes] requests. For example, if you try to retrieve a container or blob that
 doesn't exist in your Storage Account, a `404` error is returned, indicating `Not Found`
```
Comment (kaerm) [inline sdk/storage/azure-storage-blob/src/samples/README.md]: Can't comment properly - in the links section there is the same typo in the link name for SDK_README_GETTING_STARTED

File: sdk/storage/azure-storage-queue/src/samples/README.md [hunk 7]
```diff
@@ -1,21 +1,24 @@
-
-## Azure Azure Storage Queue Samples client library for Java
+---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-queue-storage
```
Comment (kaerm) [inline sdk/storage/azure-storage-queue/src/samples/README.md]: I'd also add one more line for ` - azure-storage`

File: sdk/storage/azure-storage-file-share/src/samples/README.md [hunk 8]
```diff
@@ -1,21 +1,24 @@
-
-## Azure Azure Storage File Samples client library for Java
+---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-storage
```
Comment (kaerm) [inline sdk/storage/azure-storage-file-share/src/samples/README.md]: there is also `- azure-files`

File: sdk/textanalytics/azure-ai-textanalytics/src/samples/README.md [hunk 9]
```diff
@@ -1,15 +1,20 @@
+---
+topic: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-ai-textanalytics
```
Comment (kaerm) [inline sdk/textanalytics/azure-ai-textanalytics/src/samples/README.md]: should be `- azure-cognitive-services`

**Pull request merged**